app/scripts/controllers/overview.js

```diff
@@ -64,6 +64,37 @@ angular.module('overview', ['stats', 'ngSanitize'])
       'data': [],
     };
 
+    $scope.$watch(function() { return ChecksService; }, function(data) {
+      if (data.success === true) {
+        $scope.checks = angular.copy(data.checks);
+      }
```

Comment:
what if the checks fail, eg because cluster is offline?
think the checks list should be cleared